File: python/example/simulator_server.py
"""
File: simulator_client.py
Project: example
Created Date: 10/10/2022
Author: Shun Suzuki
-----
Last Modified: 28/05/2023
Modified By: Shun Suzuki (suzuki@hapis.k.u-tokyo.ac.jp)
-----
Copyright (c) 2022-2023 Shun Suzuki. All rights reserved.

"""

from pyautd3.extra import Simulator


if __name__ == "__main__":
    Simulator().settings_path("settings.json").vsync(True).gpu_idx(0).run()

File: capi-old/example/simulator_server.c
/*
 * File: simulator_server.c
 * Project: example
 * Created Date: 10/10/2022
 * Author: Shun Suzuki
 * -----
 * Last Modified: 21/12/2022
 * Modified By: Shun Suzuki (suzuki@hapis.k.u-tokyo.ac.jp)
 * -----
 * Copyright (c) 2022 Shun Suzuki. All rights reserved.
 *
 */

#include "simulator.h"

int main(void) {
  const char* settings = "settings.json";

  AUTDExtraSimulator(settings, true, 0);

  return 0;
}
